frontend/pictonode-web/src/components/nodes/Flow.vue

```diff
@@ -165,9 +165,18 @@ function _getPipeline(): SerializedPipeline {
     const node = {
       id: vueFlowNode.data.realId,
       template: vueFlowNode.data.node.templateName,
-      metadata: {}
+      metadata: {},
+      values: {} as Record<string, any>,
     };
 
+    // Get the specific data for this item.
+    const specificData = specificDataMap.get(vueFlowNode.id);
+    if (specificData) {
+      if (specificData.type == SpecificDataType.InputImage) {
+        node.values["image"] = specificData.imageId;
```

Comment:
@parkern342 Does this line up with your implementation? I set the `"image"` field in the `values` map to the image ID.